dependencies.toml

```diff
@@ -55,10 +55,10 @@ shiro = "1.3.2"
 slf4j = "1.7.36"
 # Ensure that we use the same Snappy version as what Curator depends on.
 # See: https://github.com/apache/curator/blob/master/pom.xml
-snappy = "1.1.10.1"
+snappy = "1.1.10.4"
```

Comment:
The master branch depends on Snappy `1.1.10.4` but Curator 5.5.0 uses `1.1.7`. It seems okay because only patch versions increased.
https://github.com/apache/curator/blob/apache-curator-5.5.0/pom.xml#L108
I wanted to note because the comment insists we use the same version.

Reply:
Oops, let me revert this also. 😓